src/config/AXISFLYINGARGUSF7AIO/config.h

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Config file changes will need to be moved to https://github.com/betaflight/config repo